integration_test.sh

```diff
@@ -22,92 +22,92 @@ ERR=0
 
 # Start training
 echo "Test training"
-python3 train_model.py --train $TRAIN_PATH --dev $DEV_PATH --monitor $DEV_PATH $TRAIN_PATH --output_dir $EXPT_DIR --print_every 30 --embedding_size $EMB_SIZE --hidden_size $H_SIZE --rnn_cell $CELL --epoch $EPOCH --save_every $CP_EVERY --batch_size 6 --write-logs 'log_test'
```

Comment:
Similar here, this would give me trouble locally to run the integration tests. Maybe there is another way we could solve this?

Reply:
I'll set those back.